index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	 <!--Header element `<h1>` - This is the main headline; include your name and what this page is-->
+	 
+
+
+	 <link rel="stylesheet" type="text/css" href="styles.css">
+	<!--Title - make it meaningful--> 
+	<title>Craig Wilkosz HTML Quiz</title>
+	<h1>Craig Wilkosz HTML meaningful Quiz...</h1>
+</head>
+<body>
+
+    <nav>
+      <ul class="navbar">
+          <li><a href="index.html">Home</a></li>
+          <!--<li><a href="page2.html">page2</a></li> -->
+          <li><a href="images">page3</a></li>
+      </ul>
+    </nav>
+
+<!--Article element containing your bio-->
+	<header>
+    <h2>About Craig</h2>
+  </header>
+	  <section>
+	  	 <article>
+
+        <title>About Craig</title>
```

Comment:
Instead of using title tag here (it should only be in the head of an HTML), wrap these in a header tag.